articles/azure/asz-sco

```diff
@@ -0,0 +1,262 @@
+
+You can use your own images for non-Windows and RHEL services, where licensing stipulates that to use the VM on our platform it must be licensed (and reported back to the software vendor) by UKCloud.
+
+**Anti-virus.** We do not provide anti-virus software as part of the service.
+
+## Protective monitoring
+
+We have implemented GPG 13-aligned Protective Monitoring across the Assured and Elevated platforms at the hypervisor level and below.
+
+We don't provide Protective Monitoring services above the hypervisor (for example, for your VM) - it is your responsibility to act at this level.
+
+In line with UKCloud's SISP, we provide notification of customer-impacting security incidents. It is your responsibility to report similar incidents to us.
+
+## Platform management
+
+Users can access, manage and view the UKCloud for Microsoft Azure service, accessing only those features allowed by their role, in any of the following ways:
+
+- **Azure Stack API.** Enables the programmatic creation and management of the services inside the platform.
+
+- **Azure Stack graphical UI.** Provides a graphical interface to access the Azure Stack environment (depending on assigned permissions).
+
+- **UKCloud Portal.** Access to service reporting and incident and request management.
+
+You cannot access the underlying infrastructure. This includes (but isn't limited to) the hardware.
+
+## Service reporting
+
+**Visibility.** Customers can utilise the Azure Stack portal to configure multiple views of their estate to monitor the health of their services in real time. Including utilisation of individual components, alerts and recent operations.
+
+**Reporting** We provide a monthly service report via the UKCloud Portal. This includes a summary of platform availability and a retrospective of the interactions with UKCloud support during the previous month. Maintenance notifications and Service Status reports are also delivered through the Portal and via the UKCloud status page https://status.ukcloud.com/.
+
+**Billing.** We provide you with monthly bills covering your monthly spend.
+
+## Customer service
+
+**Customer Success Manager (CSM).** All new customers of the service have access to a UKCloud Customer Success Manager, who will help you with your adoption of the UKCloud for Microsoft Azure service,  including finding relevant systems and using UKCloud tools and processes.
+
+**Cloud Architects.** UKCloud Cloud Architects support you during the design of solutions for the cloud platform. UKCloud Cloud Architects are ideally placed to help reconcile your requirements with the UKCloud platforms. We recommend engagement with a Cloud Architect when implementing complex solutions, such as those using HybridConnect or a Walled Garden.
```

Comment:
UKCloud Cloud Architects is a bit unwieldy. Can we just use Cloud Architects instead. Or limit it to just one use of UKCloud Cloud Architects